lib/cog/commands/alias.ex

```diff
@@ -6,31 +6,38 @@ defmodule Cog.Commands.Alias do
   @moduledoc """
   Manages aliases
 
+  ## Usage
+
+    alias <subcommand>
+
+
   ## Subcommands
-  * new <alias-name> <alias-definition> -- creates a new alias visible to the creating user.
-  * mv <alias-name> <site | user>:[new-alias-name] -- moves aliases between user and site visibility. Optionally renames aliases.
-  * rm <alias-name> -- Removes aliases
-  * ls [pattern] -- Returns the list of aliases optionally filtered by pattern. Pattern support basic wildcards with '*'.
+
+    • new <alias-name> <alias-definition> -- creates a new alias visible to the creating user.
+    • mv <alias-name> <site | user>:[new-alias-name] -- moves aliases between user and site visibility. Optionally renames aliases.
+    • rm <alias-name> -- Removes aliases
+    • ls [pattern] -- Returns the list of aliases optionally filtered by pattern. Pattern support basic wildcards with '*'.
```

Comment:
I think we could probably just get rid of the dots here. Although, they do look pretty fancy.